### ISSUE TITLE
Remote sends output even though --quiet or -q is passed

### ISSUE THREAD

**Issue by Devvox93** [gitbucket/gitbucket#1721]
## Issue
**Impacted version**: 4.13.0

**Deployment mode**: with an http frontend (not sure how configured)

**Problem description**:
When pushing with the Git command line to a Gitbucket repository and thereby passing -q or --quiet to the git push command, the remote (Gitbucket) still sends output back and therefore fails to behave as expected.
What the -q and/or --quiet parameter (on the client side) does is:
- squelch the output from the sending side (i.e. local/client)
- ask "quiet" to the receiving side (i.e. remote/gitbucket), if they know how to be quiet.
That last part is what's not being handled correctly by Gitbucket, as far as I can see.

It is annoying, as Git outputs to stderr instead of stdout, causing errors and fails in continuous deployment. When specifying -q, I am expecting no output at all, unless an error occurs.

Steps to reproduce:
- have a git repository
- change something
- commit
- push with -q or --quiet parameter

Look for -q or --quiet at https://git-scm.com/docs/git-push
_Note: I am using git version 2.14.1.windows.1_

**Comment by takezoe** [comment]
Indeed, but I don't know how to fix it because it's processed in JGit...
